addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -166,6 +166,7 @@
                                 onColorLeave="(params) => this._colorpaletteApply('color', params)"
                                 onColorPicked="(params) => this._onColorpaletteColorPicked('color', params)"
                                 onCustomColorPicked="(params) => this._onColorpaletteColorPicked('color', params)"
+                                getTemplate="this.props.getColorpickerTemplate"
```

Comment:
Do we agree that it is ok to modify this XML file in stable ? From another review, I learned that apparently we can modify `static` xml so I guess it is ok here :thinking:

Reply:
AFAIK yes: those are not imported as database records, so they are always up to date.